src/helloworld.cpp

```diff
@@ -0,0 +1,7 @@
+#include <iostream>
+using namespace std;
+string welcome()
+{
+    std::cout << "Hello World" << std::endl;
```

Comment:
why cout? I think we may want some logging later for this but for now, we don't want to print the text.